Examples of Code of Conduct

ceefour opened the pull request.

ceefour committed:
Examples of Code of Conduct

zee-moz commented:
@ceefour thanks you for finding these! It's useful to see examples in Indonesian, as well! I think we'll limit it to two examples for each language, so we don't end up with an overly-long list. Can you pick the two best examples in Indonesian? Thanks! 😸

ceefour committed:
Only 2 Indonesian examples + Pedoman Pemberitaan Media Siber

@zee-moz I appeal, to have 2 Indonesian examples that are varied enough, plus Pedoman Pemberitaan Media Siber which is actually a nationwide formal cybermedia of ethics (it's not sufficient as a standalone code of conduct, yet it's also an accepted "best practices" which should be incorporated in most if not all communities, and needs better exposure).

Refs #16
